src/components/state-details.js

```diff
@@ -492,6 +492,10 @@ export default function StateDetailsPage ({ location, data }) {
                   To cut this pollution, if you have a car, your next one needs
                   to be an electric vehicle (EV).
                 </p>
+                <p className="mt-5">
+                  Or try going car free when public transit, e-bikes, or other
+                  zero mobility options are available.
```

Comment:
This reads a little weird to me. I'm not sure what 'zero mobility' is. 

Maybe "Or try going car-free with public transit, bikes/e-bikes, or walking if it works for you."

Reply:
Oh oops, that was supposed to say "zero emissions mobility options" but that's basically as wonky as it gets (I'm trying to convey things like e-moped, boost boards, all that stuff). I'll use your version.